src/RoatpCourseManagement/README.md

```diff
@@ -0,0 +1,54 @@
+## ⛔Never push sensitive information such as client id's, secrets or keys into repositories including in the README file⛔
+
+# _Roatp Course Management Outer API_
+
+<img src="https://avatars.githubusercontent.com/u/9841374?s=200&v=4" align="right" alt="UK Government logo">
+
+[![Build Status](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_apis/build/status%2FAPIM%2Fdas-apim-endpoints-RoatpCourseManagement?repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=master)](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_build/latest?definitionId=2880&repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=master)
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkillsFundingAgency_das-apim-endpoints_RoatpCourseManagement&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkillsFundingAgency_das-apim-endpoints_RoatpCourseManagement)
```

Comment:
This is not showing up, perhaps due to bad link, use this link below

[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkillsFundingAgency_das-apim-endpoints_RoatpCourseManagement&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkillsFundingAgency_das-apim-endpoints_RoatpCourseManagement)